prisma/migrations/20210529200211_add_tables_for_nextjs_auth/migration.sql

```diff
@@ -0,0 +1,77 @@
+-- CreateTable
+CREATE TABLE "Account" (
```

Comment:
Can you also explain what the difference between an `Account` entity and a `User` entity is? Is there an issue with having the fields in `User` be in Account?

Reply:
That's a good question - I was just following what the [documentation](https://next-auth.js.org/adapters/prisma) suggested. I don't think Account and Session are required though - do you think I should remove them?